superset/assets/webpack.config.js

```diff
@@ -108,6 +108,14 @@ const config = {
         include: APP_DIR + '/node_modules/mapbox-gl/js/render/painter/use_program.js',
         loader: 'transform/cacheable?brfs',
       },
+      {
+        test: /\.po$/,
+        loader: 'po-catalog-loader',
```

Comment:
Is babel or `po-catalog-loader` collecting the strings? I thought the babel command was doing it. Do we need both?